_sections/home-english.md

```diff
@@ -196,6 +220,9 @@ For multiple-choice questions, surveys often include an option of marking **othe
 
 Click on the link to look at the complete [pizza_questionnaire](https://docs.google.com/spreadsheets/d/1y9LcFUaJ_MDRpqbzHVxkD_k6YzSQCllqh3Excy4iffg/edit?usp=sharing).
 
+**Caveat**
+When you export data using this **or_other** option, in the **favorite_topping** column, you will see a value  **other**. A separate column will have the answer for the questions in which the user selected **other**. This makes data analysis more cumbersome, so we do not recommend the **or_other** construct for large scale data collection efforts. Therefore, we recommend using the[relevant](#relevant) function for specifying the other option.
```

Comment:
This is meant to give more details on how the data is exported when a user decides to use automatic `or_other` 

> We don't currently support open selects so I don't think there's any alternative to what's mentioned here.

When you say we don't support does it mean it can be excluded entirely from the docs?
I tested this on Ona aggregate servers and it works as documented here

Reply:
There's no difference between using the `or_other` shortcut vs using relevance manually, right? This caveat says that using `or_other` is worse than using relevance directly for analysis but I don't think that's true.

Open selects would be where you can add a value that's not in the list but that's stored in the same field. That's not supported. https://forum.getodk.org/t/allow-custom-choices-for-selects-that-get-saved-to-the-same-field-xforms-open-selects/22788

Reply:
I get what you mean and it is a fair comment. Testing it again it works similar to other but the issue I have noticed is when one is using Ona servers and connect data to Tableau using their connector which does not work. Otherwise it works just fine with data exports. It will make sense to remove it